Recolor.php

```diff
@@ -53,8 +53,12 @@ public function image($imageName, $newFilename)
             }
         }
 
-        imagepng($image, $this->imageFolder . $newFilename);
-        imagedestroy($image);
+        if (imagepng($image, $this->imageFolder . $newFilename)) {
+            imagedestroy($image);
+            return "New image was created and saved.";
+        }
+
+        return "Could not create a new image.";
```

Comment:
This isn't wrapped in an `else{` because the above code has already returned, that means that the control of the code has already moved back to whatever called the method. So if the code has reached this point, we know the new image wasn't created, so we can assume a fail here.